.gitlab-ci.yml

```diff
@@ -0,0 +1,274 @@
+stages:
+  - lint
+  - check
+  - test
+  - build
+  - publish
+
+workflow:
+  rules:
+    - if: $CI_COMMIT_TAG
+    - if: $CI_COMMIT_BRANCH
+
+variables:                         &default-vars
+  GIT_STRATEGY:                    fetch
+  GIT_DEPTH:                       100
+  CARGO_INCREMENTAL:               0
+  ARCH:                            "x86_64"
+  CI_IMAGE:                        "paritytech/bridges-ci:production"
```

Comment:
[This](https://github.com/paritytech/scripts/blob/master/.gitlab-ci.yml#L375-L384) is how the image updates.
![image](https://user-images.githubusercontent.com/17856421/120534712-70f40e80-c3e2-11eb-95d7-9cea4073a23b.png)
1. the new CI image is built at 6 AM nightly and pushed with `staging` tag.
2. `staging` is tested against `bridges` CI on master and gets a vuln scan
3. if green, it gets into production

in this case it's failed because Gitlab CI is not in master yet.